webui/react/src/dev.ts

```diff
@@ -0,0 +1,18 @@
+/* Tools and tweaks for dev environments */
+
+import * as Api from 'services/api';
+import { globalStorage } from 'utils/storage';
+
+const setServerAddress = (address: string) => {
+  globalStorage.set('server-address', address);
+  Api.updateDetApi({ basePath: address });
+};
+
+window.dev = window.dev || {};
+window.dev.setServerAddress = setServerAddress;
+window.dev.resetServerAddress = () => globalStorage.remove('server-address');
+
+if (process.env.IS_DEV) {
+  window.dev.api = Api;
+  document.title = `[dev] ${document.title}`;
```

Comment:
**suggestion:** I personally think this is unnecessary just because we have the blue favicon and it will show up in the history dropdown (when you click and hold the back button). Not having the `[dev] ` also just gives us a little bit more title space back for other info.